tldw_Server_API/app/api/v1/endpoints/auth.py

```
@@ -244,6 +309,272 @@ def _normalize_magic_email(email: str) -> str:
+        refresh_token=refresh_token,
+        token_type="bearer",
+        expires_in=settings.ACCESS_TOKEN_EXPIRE_MINUTES * 60,
+    )
+
+
+@router.get("/federation/{provider_slug}/login")
+async def federation_login(
+    provider_slug: str,
+    request: Request,
+    org_id: int | None = Query(None, ge=1),
+    session_manager: SessionManager = Depends(get_session_manager_dep),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)